projects/ngx-quill/src/lib/quill-view-html.component.ts

```diff
@@ -37,19 +37,21 @@ export class QuillViewHTMLComponent implements OnChanges {
   private sanitizer = inject(DomSanitizer)
   private service = inject(QuillService)
 
-  ngOnChanges(changes: SimpleChanges) {
-    if (changes.theme) {
-      const theme = changes.theme.currentValue || (this.service.config.theme ? this.service.config.theme : 'snow')
-      this.themeClass.set(`ql-${theme} ngx-quill-view-html`)
-    } else if (!this.theme()) {
-      const theme = this.service.config.theme ? this.service.config.theme : 'snow'
-      this.themeClass.set(`ql-${theme} ngx-quill-view-html`)
-    }
-    if (changes.content) {
-      const content = changes.content.currentValue
-      const sanitize = [true, false].includes(this.sanitize()) ? this.sanitize() : (this.service.config.sanitize || false)
+  constructor() {
+    toObservable(this.theme).subscribe((newTheme) => {
```

Comment:
Probably you can use an `effect`?

If not, do we need to have an `unsubscribe`?